README.md

```diff
@@ -3,48 +3,51 @@
 API plugins, microservices, and commands for accessing NetCDF data/metadata contained in iRODS data objects.
 
 Currently supported version(s) of iRODS:
-   - 4.2.8
+   - 4.3.1
+   - 4.2.12
 
 OS Distributions presently supported:
+* on iRODS 4.3.1
+   - AlmaLinux 8 (RHEL 8)
+   - Centos 7
+   - Debian 11
+   - Debian 12
+   - Rocky Linux 9 (RHEL 9)
+   - Ubuntu 18.04
+   - Ubuntu 20.04
+   - Ubuntu 22.04
+* on iRODS 4.2.12
    - Centos 7
-   - Ubuntu 16.04
    - Ubuntu 18.04
```

Comment:
This list can be removed since it depends specifically on the iRODS server.

I think it would be better to state which OS this is NOT supported on.

Reply:
I guess to mention the "not supported on" thing would simply be to point out we've dropped support for Ubuntu 16.04 in iRODS 4.2.12 and later, but of course the world knows it by now, so I'll jjust delete for now.  Or have I misunderstood the recommendation? In case that isn't what we wanted, I'll keep the commit distinct from the others for now.